jans-lock/cedarling/rustfmt.toml

```diff
@@ -1,2 +1,72 @@
-hard_tabs = true
+edition = "2021"
+unstable_features = true
+version = "Two"
```

Comment:
Version "Two"? What is the version of exactly? Why is it "Two"?

Reply:
https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#version
here is link to the config documentation

Reply:
https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#version
here is link to the config documentation